src/main/java/com/sergiofah/service/ProductService.java

```diff
@@ -0,0 +1,38 @@
+package com.sergiofah.service;
+
+import com.sergiofah.dto.ProductDTO;
+import com.sergiofah.model.Product;
+import com.sergiofah.repository.ProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ProductService {
+    @Autowired
+    private ProductRepository productRepository;
+    public List<ProductDTO> getProductFromCategoryId(Long id){
```

Comment:
Method declaration shall be separated by one line from previous method or attribute declaration.